.github/workflows/deploy.yml

```diff
@@ -0,0 +1,36 @@
+name: Deploy MkDocs site to Pages
+
+on:
+  push:
+    branches: [$default-branch, deploy-to-gh-pages]
```

Comment:
guessing you'll remove the second branch before merging?

Reply:
Yep, should do :-) - impossible to test without tho!